tests/unit/payments.js

```diff
@@ -165,6 +155,24 @@ define('tests/unit/payments',
             });
         });
 
+        it('defines a configurable adapter', function(done) {
+            var adapterConfigured = false;
+
+            setupMocks().mock('fxpay', {
+                init: function(opt) {
+                    // This simulates how fxpay will configure
+                    // the Marketplace adapter when fxpay.init()
+                    // is called.
+                    opt.adapter.configure({});
```

Comment:
Oh, I guess you're testing that the `configure` method is defined so that this doesn't blow up. I think `sinon` will complain if you attach a spy and there's no function on what you're attaching to. But it might make more sense to call it as you have here.